groups/managers.py

```diff
@@ -1,9 +1,47 @@
+import datetime
+
+from django.apps import apps
 from django.contrib.auth import get_user_model
 from django.db import models
 from polymorphic import PolymorphicManager, PolymorphicQuerySet
 
 
-class GroupQuerySet(models.QuerySet):
+DEFAULT_WITHIN_DAYS = apps.get_app_config('groups').default_within_days
+
+
+class WithinDaysUserQuerySetMixin:
+    """
+    A mixin that adds methods for returning items that have recently been posted (to).
+
+    Can be mixed into a Manager or a QuerySet.
+
+    By default, is intended for use with User objects.  To change this, override the
+    condition used in the `since()` method.
+    """
+    @staticmethod
+    def get_threshold_delta(timedelta):
+        """Return the earliest posting *time* a comment can have and still be recent."""
+        return datetime.datetime.now() - timedelta
+
+    @staticmethod
+    def get_threshold_date(within_days=DEFAULT_WITHIN_DAYS):
+        """Return the earliest posting *date* a comment can have and still be recent."""
+        return datetime.date.today() - datetime.timedelta(days=within_days)
+
+    def within_days(self, days=DEFAULT_WITHIN_DAYS):
+        """All users that created a comment within the last `days` days."""
+        return self.since(self.get_threshold_date(days))
+
+    def within_time(self, timedelta):
+        """All users that created a comment within the last `timedelta`."""
+        return self.since(self.get_threshold_delta(timedelta))
+
+    def since(self, when):
+        """All the comments belonging to items in this queryset posted since `when`."""
+        return self.filter(comments__date_created__gte=when).distinct()
```

Comment:
This filter doesn't seem generic. Perhaps 

```python
filter = {self.since_filter: when}
return self.filter(**filter).distinct()
```
That way, the other `since` methods can go, and each can have something like this set:

```
class MyQuerySet(...):
    since_filter = 'comments__date_created__gte'
```